### DISCUSSION TITLE
Admonition (Title) in TOC

### DISCUSSION THREAD

**Discussion by arnold518** [squidfunk/mkdocs-material#5456] [Q&A] [answered]
Is there a way to include a single admonition itself to the Table Of Contents?
Otherwise, can I add header attribute to the admonition title, so that it can be included in the TOC?

If there is not any way to do this, can I use an invisible header that is only visible in TOC, or a visible header that is not visible in TOC?

**Comment by RossKen** [comment]
I also think this would be a great feature!

**Comment by feasgal** [comment, marked as answer]
I managed to get this effect in a FAQ by using invisible headings that have the same text as the admonition's title. The FAQ entries are all in `details` admonitions that are collapsed by default.

1.    In `extra.css`, I [created a custom admonition class](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#custom-admonitions).
2.    Also in `extra.css`, I styled `<h5>` elements that are inside that custom admonition class with `visibility: hidden;`.
```
details.ilokfaq > h5 {
    visibility: hidden;
```
3. In the markdown, I put a `#####` inside the `details`, with the same text as the admonition's `title`. Here's a real example with some of the text content trimmed out for brevity.
```
??? ilokfaq "What is an iLok USB?"
	
	##### What is an iLok USB? { #what_ilok }
	
	The iLok USB is a key device that holds licenses for [participating software products](https://ilok.com/#!ilok-publishers). A single iLok USB can store up to 1500 separate licenses (3rd generation), 500 separate licenses (2nd generation), or 118 separate licenses (1st generation).
```

See the result [here](https://help.ilok.com/faq_ilok.html#what_ilok).

Note: There's also some `extra.js` in play on that site, to make all the parent `details` elements expand if the destination anchor is inside them. I talked about how I did that [here](https://github.com/squidfunk/mkdocs-material/issues/4125#issuecomment-1275400860).
